clients/shared/utils.go

```diff
@@ -44,8 +44,7 @@ func DefaultValue(column columns.Column, dialect sql.Dialect, additionalDateFmts
 			return nil, fmt.Errorf("colVal is not type *decimal.Decimal")
 		}
 
-		// TODO: Call [String] instead.
-		return val.Value(), nil
+		return val.String(), nil
```

Comment:
This is the only place it was used.

Reply:
And this function is only called when backfilling.

Reply:
@Tang8330 I feel like since we were sometimes emitting a string before, always emitting a string should be fine?

Reply:
Yeah it's fine, I just tested it

Reply:
Sweet thanks!